.github/workflows/docker.yaml

```diff
@@ -45,6 +45,10 @@ jobs:
       run: |
         yum -y upgrade
         yum -y install git-core gcc make m4 patch tar unzip perl perl-Test-Simple perl-IPC-Cmd xz gcc-c++ dejagnu
+        # To avoid linking against libdb and gdmb libraries on Amazon Linux 2.
```

Comment:
Note that usually you can run something like `./configure --without-db --without-gdbm` and the Python build process should not enalbe those libraries.

The default behaviour is to look for any librarie available via `pkg-config` and enable it by default.

But it should be possible to run `./configure` and ingore some of the available libraries.

Reply:
Thanks for the suggestion! I've already checked configure flags and couldn't find something relevant.

Previously, we were patching `DISABLED_MODULES` list in Python sources, but this is no longer present in Python 3.13 sources.

For the sake of uniformity, I went for what we already do on macOS.

Reply:
On a second thought, a cleaner solution would be to leverage the new `PYTHIA_BUILD_TESTS` env var to only enable these tests when building through GitHub.

This way, when building locally, you don't need to have `perl-IPC-Cmd` installed for tests during building on generic Linux. Therefore, you don't need `libdb` and `gdmb` headers.

Reply:
This hack is only applied through GHA, so I would leave local builds as-is.

Reply:
Can we make sure that the "-dev" version of the packages are not installed ?

For example, if `libnsl-dev` is not installed, in theory, Python would not be able to build it's `nis` module